### ISSUE TITLE
Refusing sql connection: too many clients already

### ISSUE THREAD

**Issue by giuseppetrotta** [rapydo/http-api#351]
It seems that sql connections are not being properly managed (or better closed).

Backend logs:
```
2023-06-20 11:02:01,249 [ERROR restapi.connectors.sqlalchemy:336] (psycopg2.OperationalError) connection to server at "sql.dockerized.io" (172.18.0.7), port 5432 failed: FATAL:  sorry, too many clients already

(Background on this error at: https://sqlalche.me/e/20/e3q8)
2023-06-20 11:02:01,255 [ERROR restapi.decorators:454] Backend database is unavailable
```
Postgres logs:
```
2023-06-20 11:00:54.919 UTC [18343] 172.18.0.3 FATAL:  sorry, too many clients already
2023-06-20 11:01:04.821 UTC [18344] 172.18.0.3 FATAL:  sorry, too many clients already
2023-06-20 11:01:06.988 UTC [18345] 172.18.0.3 FATAL:  sorry, too many clients already
```
So after a while (depends on max connections config) the Backend becomes unusable because it can't access the db.

NOTE: this happens ONLY with rapydo 3.0 (using postgreSQL db)

**Comment by mdantonio** [comment]
Just pushed a temptative fix, please let me know if it addresses your issue 🙏

**Comment by giuseppetrotta** [comment]
Hi @mdantonio . First of all thank you for the very quick response!

I did some tests and although the service is more stable I'm still not convinced that the connections are released correctly. The test done is the following:

I reduced the **verification time** to just 10 seconds (ALCHEMY_VERIFICATION_TIME = 10) and launched a series of requests with access to the db, in particular `get_user` invoked by the `auth/profile` endpoint
```bash
for i in `seq 1 100`; do https -A bearer -a <token> <base_url>/auth/profile; done
```
From a starting value of about 60 active connections after some tests the value arrived at
```sql
SELECT count(*) FROM pg_stat_activity;
181
```
If the initial cause was due to releasing connections with expired verification time, maybe there is still something to fix. Or am I doing something wrong in the test?

ty dude
